docs/r2/image_summaries.ipynb

```diff
@@ -0,0 +1,736 @@
+      ]
+    },
+    {
+      "metadata": {
+        "id": "IJNpyVyxbVtT",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "# Clear out any prior log data.\n",
+        "!rm -rf logs\n",
+        "\n",
+        "# Sets up a timestamped log directory.\n",
+        "logdir = \"logs/train_data/\" + datetime.now().strftime(\"%Y%m%d-%H%M%S\")\n",
+        "# Creates a file writer for the log directory.\n",
+        "file_writer = tf.summary.create_file_writer(logdir)\n",
+        "\n",
+        "# Using the file writer, log the reshaped image.\n",
+        "with file_writer.as_default():\n",
+        "  tf.summary.image(\"Training data\", img, step=0)"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "rngALbRogXe6",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "Now, use TensorBoard to examine the image. Wait a few seconds for the UI to spin up."
+      ]
+    },
+    {
+      "metadata": {
+        "id": "T_X-wIy-lD9f",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "%tensorboard --logdir logs/train_data"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "c8n8YqGlT3-c",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "<img class=\"tfo-display-only-on-site\" src=\"https://github.com/tensorflow/tensorboard/blob/master/docs/r2/images/images_single.png?raw=1\"/>"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "34enxJjjgWi7",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "The \"Images\" tab displays the image you just logged. It's an \"ankle boot\". \n",
+        "\n",
+        "The image is scaled to a default size for easier viewing. If you want to view the unscaled original image, check \"Show actual image size\" at the upper left. \n",
+        "\n",
+        "Play with the brightness and contrast sliders to see how they affect the image pixels."
+      ]
+    },
+    {
+      "metadata": {
+        "colab_type": "text",
+        "id": "bjACE1lAsqUd"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "## Visualizing multiple images\n",
+        "\n",
+        "Logging one tensor is great, but what if you wanted to log multiple training examples?\n",
+        "\n",
+        "Simply specify the number of images you want to log when passing data to ```tf.summary.image()```."
+      ]
+    },
+    {
+      "metadata": {
+        "id": "Fr6LFQG9UD6z",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "<img class=\"tfo-display-only-on-site\" src=\"https://github.com/tensorflow/tensorboard/blob/master/docs/r2/images/images_multiple.png?raw=1\"/>"
```

Comment:
I think we should put the image right under the corresponding '%tensorboard' (in this case it is a bit further down for the multiple image, but I also see the images_arbitrary image twice).

Reply:
Fixed